core/src/main/java/nl/nn/adapterframework/pipes/CompressPipe.java

```diff
@@ -232,27 +216,21 @@ public void setResultIsContent(boolean b) {
 		resultIsContent = b;
 	}
 
-	public String getZipEntryPattern() {
-		return zipEntryPattern;
-	}
-
 	@IbisDoc({"the pattern for the zipentry name in case a zipfile is read or written", ""})
 	public void setZipEntryPattern(String string) {
 		zipEntryPattern = string;
 	}
 
-	public boolean isConvert2String() {
-		return convert2String;
-	}
-
-	@IbisDoc({"if <code>true</code> result is returned as a string, otherwise as a byte array", "false"})
+	@Deprecated
+	@ConfigurationWarning("It should not be necessary to specify convert2String. If you encounter a situation where it is, please report to Frank!Framework Core Team")
+	@IbisDoc({"if <code>true</code> result is returned as character data, otherwise as binary data", "false"})
 	public void setConvert2String(boolean b) {
 		convert2String = b;
 	}
 
 	@IbisDoc({"when set to gz, the gzip format is used. when set to another value, the zip format is used. if not set and direction is compress, the resultiscontent specifies the output format used (resultiscontent=<code>true</code> -> gzip format, resultiscontent=<code>false</code> -> zip format) if not set and direction is decompress, the messageiscontent specifies the output format used (messageiscontent=<code>true</code> -> gzip format, messageiscontent=<code>false</code> -> zip format)", ""})
-	public void setFileFormat(String string) {
-		fileFormat = string;
+	public void setFileFormat(FileFormat format) {
+		fileFormat = format;
```

Comment:
Please also document the FileFormat enum values, and remove the corresponding documentation from the  setter-IbisDoc